src/main/java/com/jfrog/ide/idea/ui/webview/WebviewObjectConverter.java

```diff
@@ -0,0 +1,213 @@
+package com.jfrog.ide.idea.ui.webview;
+
+import com.jfrog.ide.common.tree.Artifact;
+import com.jfrog.ide.common.tree.ImpactTreeNode;
+import com.jfrog.ide.common.tree.Issue;
+import com.jfrog.ide.common.tree.LicenseViolation;
+import com.jfrog.ide.idea.ui.webview.model.*;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class WebviewObjectConverter {
```

Comment:
The methods in this file do not comply with our standards.
I suggest doing the following:
1.  Splitting long methods into smaller methods
2. Using "for in" loops over old style "for int" loops.
3. Use apache-commons libraries when possible